mpas_analysis/ocean/time_series_antarctic_melt.py

```diff
@@ -0,0 +1,399 @@
+            self.plot(config, timeSeries, obsCount, obsMeltRate, obsMeltRateUnc, title, xLabel, yLabel, figureName )
+
+
+    def plot(self, config, modelValues, obsCount, obsMean, obsUncertainty, title, xlabel, ylabel, fileout):
+
+        """
+        Plots the list of time series data sets and stores the result in an image
+        file.
+        Parameters
+        ----------
+        config : instance of ConfigParser
+            the configuration, containing a [plot] section with options that
+            control plotting
+
+        modelValues : xarray DataSet
+            the data set to be plotted
+
+        obsMean : float
+            a single observed mean value to plot as a constant line
+
+        obsUncertainty : float
+            The observed uncertainty, to plot as a shaded rectangle around the mean 
+
+        title : str
+            the title of the plot
+
+        xlabel, ylabel : str
+            axis labels
+
+        fileout : str
+            the file name to be written
+
+        Authors
+        -------
+        Xylar Asay-Davis, Stephen Price
+        """
+
+        # play with these as you need
+        figsize = (15, 6)
+        dpi = 300
+        modelLineStyle = 'b-'
+        modelLineWidth = 2
+        obsColor = 'gray'
+        obsLineWidth = 1 
+
+        maxXTicks = 20
+        calendar = self.calendar
+
+        axis_font = {'size': config.get('plot', 'axisFontSize')}
+        title_font = {'size': config.get('plot', 'titleFontSize'),
+                      'color': config.get('plot', 'titleFontColor'),
+                      'weight': config.get('plot', 'titleFontWeight')}
+
+        plt.figure(figsize=figsize, dpi=dpi)
+
+        minDays = modelValues.Time.min()
+        maxDays = modelValues.Time.max()
+        plt.plot(modelValues.Time.values, modelValues.values, modelLineStyle,
+                 linewidth=modelLineWidth, label='model')
+
+        ax = plt.gca()
+
+        # this makes a "patch" with a single rectangular polygon, where the pairs are time and melt 
+        # rate for the 4 corners, and "True" means it is a closed polygon:
+#        patches = [Polygon([[minDays, obsMean-obsUncertainty], [maxDays, obsMean-obsUncertainty],
+#                        [maxDays, obsMean+obsUncertainty], [minDays, obsMean+obsUncertainty]])]
+
+        # make the polygon gray and mostly transparent
+#        p = PatchCollection(patches, color=obsColor, alpha=0.15)
+        # add it to the plot on the current axis
+#        ax.add_collection(p)
+
+        # also plot a line
+#        plt.plot([minDays, maxDays], [obsMean, obsMean], color=obsColor, linewidth=obsLineWidth)
+
+	# plot error bars rather than the "patch" used above
+        for iObs in range( obsCount ):
+	    plt.errorbar( ((maxDays - minDays)/4)*(iObs+1)+minDays, obsMean[iObs], yerr=obsUncertainty[iObs], fmt='o', ecolor='k', 
+	    		capthick=2, label='obs{}'.format(iObs+1))
+
+        # add legend
+	plt.legend( loc='lower right' )
```

Comment:
@stephenprice, if you add numpoints=1 to this call, you should (hopefully) get only one point per obs.  It might screw up the line in the legend, not sure about that, but I hope not.

Reply:
That worked. Thanks! So here's where we are now ... I think this is more-or-less working as we want it to, at least for these sets of figures. There are a few more issues w/ how other datasets get formatted to work here, and what we do w/ missing data, etc. but those are pretty minor. The next thing I will start thinking about is if / how we can use the existing data in the scope of this same chunk of code to spit out the bar chart type plots we discussed above. Unless you think that should go in it's own set of code (which would duplicated a lot of what is already included here)?

![melt_flux_ronne](https://user-images.githubusercontent.com/4983771/32024084-90270b98-b998-11e7-9173-ebca93d5aee8.png)